src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+$card-width: 200px;
+$card-height: 406px;
+$image-width: $card-width - 40px;
+$image-height: $card-height - 272px;
+
+body {
+  margin: 0;
+  font-family: 'Roboto', serif;
+  background-color: gray;
+}
+
+.card {
+  width: $card-width;
+  height: $card-height;
+  box-sizing: border-box;
+  border-radius: 5px;
+  background-color: #fff;
+}
+.image {
+  display: block;
+  width: $card-width - 40px;
+  height: $card-height - 272px;
+  margin: 0 19px;
+  padding-top:  $card-height - 374px;
+}
+.content__description{
+  position: relative;
+  margin: 40px 16px 0 16px ;
+}
+
+.description--name{
+  font-size: 12px;
+  font-weight: 500;
+}
+.description--code{
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+  margin-top: 4px;
+}
+.content__price{
+  margin: 24px 16px 0 16px;
+  display: flex;
+  justify-content: space-between;
+}
+.price--label{
+  color: #616070;
+  font-size: 12px;
+  font-weight: 400;
+  text-align: center;
+}
+.price--numbers{
+  font-family: 'Roboto', serif;
+  font-weight: 700;
+  font-size: 16px;
+}
+.button__link{
+text-decoration: none;
+font-size: 14px;
+font-weight: 700;
+color: #fff;
+}
+.button{
+  color: #FFFFFF;
+  font-size: 14px;
+  font-weight: 700;
+  display: block;
+  box-sizing: border-box;
+  height: 40px;
+  width: 166px;
+  margin: 0 auto;
+  margin-top: 16px;
+  border-radius: 5px;
+  background-color: #00ACDC;
+  border: none;
+}
+.reviews{
+  text-align: center;
+  line-height: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  color: #060B35;
+}
+.button__link:hover{
+  color: #00ACDC;
+}
```

Comment:
The hover effect for `.button__link` changes the text color to `#00ACDC`, which might conflict with the hover effect of `.button` if they are used together. Ensure that the hover styles are consistent with the intended design.